netbox/utilities/templates/builtins/copy_content.html

```diff
@@ -0,0 +1,3 @@
+<a class="btn btn-sm {{ color }} copy-content" data-clipboard-target="{{ target }}" title="Copy to clipboard">
+  <i class="mdi mdi-content-copy"></i>
+</a>
```

Comment:
nit: happened to notice that this change seems to make the file miss the final newline?